build.sh

```diff
@@ -220,6 +220,30 @@ case "$COMMAND" in
 esac
 export CONFIGURATION
 
+xcode_for_swift() {
```

Comment:
Does this do the right thing if multiple Xcode 7 betas are found? It doesn't appear to do anything clever like preferring the most recent version, so it will just pick whichever is first in Spotlight's results. Do we care?